.yarn/plugins/@rnx-kit/yarn-plugin-dynamic-extensions.cjs

```diff
@@ -0,0 +1,96 @@
+// @ts-check
+
```

Comment:
This file should be ignored except for informational purposes. It is the bundle produced by rnx-kit.